src/azurelinux/3.0/net9.0/cross/amd64-sanitizer/Dockerfile

```diff
@@ -6,7 +6,7 @@ ARG ROOTFS_DIR=/crossrootfs/x64
 RUN /scripts/eng/common/cross/build-rootfs.sh x64 bionic --skipunmount
 
 RUN TARGET_TRIPLE="x86_64-linux-gnu" && \
-    CLANG_MAJOR_VERSION=$(clang --version | grep -oP "(?<=version )\d+") && \
```

Comment:
Is the switch from `\d` to `0-9` just because we need to include `.`? And does `.` need to be escaped in this form?